src/gmail.js

```diff
@@ -2567,25 +2567,25 @@ var Gmail = function(localJQuery) {
 
 
     api.check.is_conversation_view = function() {
-        var flag_name = "bx_vmb";
-        var flag_value = undefined;
-
-        var array_with_flag = api.tracker.globals[17][4][1];
-
-        for (var i = 0; i < array_with_flag.length; i++) {
-            var current = array_with_flag[i];
-
-            if (current[0] === flag_name) {
-                flag_value = current[1];
-
-                break;
+        if( window.gmail.check.is_new_gui() ) {
```

Comment:
Oh. This line is actually *wrong*. My bad for not correcting it.

I will fix it tomorrow and issue an update as 0.7.5.

Reply:
I have created a small PR #496 for this. I have tested it locally. If you want you can review, approve and release it. 

Thanks